balancer/frontend-v2
# Hotfix: Allow staking of Avalanche FX pool

# Description

Currently there is no way to stake tokens for the Avalanche FX pool (`0x55bec22f8f6c69137ceaf284d9b441db1b9bfedc000200000000000000000011`) via the UI because of two reasons:
- The staking card is currently hidden when it is a custom pool type that has a custom way of adding liquidity (FX/Gyro)
- The pool wasn't in the staking approve list, which made the stake button not appear in the porfolio page. 

This fixes both those issues.

I did notice the Gyro pools aren't on the staking lists, should they be so that users can stake their LBP through the Balancer UI?

## Type of change


## How should this be tested?

- Ensure the add liquidity and stake cards appear as normal on normal pools (e.g. /#/avalanche/pool/0x054e7b0c73e1ee5aed6864fa511658fc2b54bcaa000000000000000000000015)
- Ensure the stake card doesn't appear on a pool without staking (e.g. /#/avalanche/pool/0x3396cca90c9335565ecdbae3a260dafb13a7959e000200000000000000000018)
- Ensure the custom card + staking appears on a custom pool (e.g. /#/avalanche/pool/0x55bec22f8f6c69137ceaf284d9b441db1b9bfedc000200000000000000000011)
- Ensure no cards appear on LBP pools (e.g. /#/ethereum/pool/0x84d9288a62132b9e9097aaefd2caea7d5ce966c50002000000000000000005ac)

## Visual context

![staking-card](https://github.com/balancer/frontend-v2/assets/525534/88911c9a-adb2-4f3f-92f3-30dffb7381b5)

## Checklist:
